pkg/pillar/pubsub/util.go

```diff
@@ -24,7 +24,7 @@ import (
 // Golang doesn't have support for a deep copy. Once can build it
 // oneself using reflect package, but it ends up doing the same thing
 // as json encode+decode apart from the exported fields check.
-func deepCopy(log *base.LogObject, in interface{}) interface{} {
```

Comment:
I cannot put the test into the `pubsub` package as I am running into an import cycle otherwise:
```
package github.com/lf-edge/eve/pkg/pillar/pubsub
	imports github.com/lf-edge/eve/pkg/pillar/types
	imports github.com/lf-edge/eve/pkg/pillar/pubsub: import cycle not allowed in test
```

Reply:
Ok